File: SDKs/amazonaws.com/rds/2014-10-31/typescript/src/sdk/models/shared/accountattributesmessage.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { AccountQuotaList } from "./accountquotalist";



// AccountAttributesMessage
/** 
 * Data returned by the <b>DescribeAccountAttributes</b> action.
**/
export class AccountAttributesMessage extends SpeakeasyBase {
  @SpeakeasyMetadata({ elemType: AccountQuotaList })
  accountQuotas?: AccountQuotaList[];
}

File: SDKs/amazonaws.com/rds/2014-10-31/python/src/sdk/models/shared/accountattributesmessage.py
import dataclasses
from typing import Optional
from ..shared import accountquotalist as shared_accountquotalist


@dataclasses.dataclass
class AccountAttributesMessage:
    r"""AccountAttributesMessage
    Data returned by the <b>DescribeAccountAttributes</b> action.
    """
    
    account_quotas: Optional[list[shared_accountquotalist.AccountQuotaList]] = dataclasses.field(default=None)
    
